docker/docker-compose.yml

```diff
@@ -90,3 +90,15 @@ services:
      - tanner_api
      - tanner_web
      - tanner_phpox
+
+# Template Injection Emulator
```

Comment:
Here docker compose is used to setup tanner as a whole. I think you should update `docker_helper.py` and add functionality related to [build](https://docker-py.readthedocs.io/en/stable/images.html#docker.models.images.ImageCollection.build), currently, we have only pull related [method](https://github.com/mushorg/tanner/blob/master/tanner/utils/docker_helper.py#L18).

Reply:
Actually, my next task is replacing docker with aiodocker, so changing `docker_helper` is of no use as we will change it anyway. For now maybe lets just keep as is?

Reply:
I don't think it will be a lot of work, just see if a dockerfile is passed in method and build if it is else pull. And with aiodocker also, I'm sure similar methods will be present.

Reply:
Cool, I'll give it a shot.